cognite/extractorutils/unstable/core/restart_policy.py

```diff
@@ -0,0 +1,29 @@
+from typing import Callable
+
+from cognite.extractorutils.unstable.core.tasks import ContinuousTask, Task
+
+RestartPolicy = Callable[[Task], bool]
```

Comment:
Should this accept the exception as well? Might make sense to only restart on certain exceptions, maybe?

Reply:
good idea